### PR TITLE
use suspense when fetching pool data from Dune

### PR DESCRIPTION
**Pull request by ribeirojose**

### CODE REVIEW

File: apps/balancer-tools/src/app/apr/round/(components)/PoolsCards.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+interface DunePoolData {
+  symbol: string;
+  pct_votes: number;
+  votes: string;
+}
+
+function PoolCard({
+  data: { symbol, pct_votes: pctVotes, votes },
+}: {
+  data: DunePoolData;
```
Comment (Rawallon) [inline apps/balancer-tools/src/app/apr/round/(components)/PoolsCards.tsx:15]: Any reason why we didn't import it from lib/dune.ts?

File: apps/balancer-tools/src/app/apr/round/(components)/PoolCard.tsx
Comment (Rawallon) [inline apps/balancer-tools/src/app/apr/round/(components)/PoolCard.tsx:1]: Should we remove this component and keep it all on PoolsCards.tsx?

File: apps/balancer-tools/src/app/apr/round/[roundId]/page.tsx
Comment (Rawallon) [inline apps/balancer-tools/src/app/apr/round/[roundId]/page.tsx:1]: Thanks a bunch, this helped me get how suspense work

File: apps/balancer-tools/src/app/apr/round/(components)/PoolsCards.tsx (hunk 1 above)
Comment (ribeirojose) [reply apps/balancer-tools/src/app/apr/round/(components)/PoolsCards.tsx:15]: No 😆 I didn't kow we had this there tbh

**Pull request merged**